truss/truss_handle.py

```diff
@@ -298,6 +298,12 @@ def _run_docker(gpus: Optional[str] = None):
                             f"src={str(LocalConfigHandler.bptr_data_resolution_dir_path())}",
                             "target=/bptr",
                         ],
+                        [
```

Comment:
for my own edification, what does this help accomplish?

Reply:
we mimic the mounted configmap directory (which is `/etc/b10_dynamic_config/environment` on model pods) here so we can make changes to `/etc/b10_dynamic_config/environment` in the integration tests to mimic patches to the `environment` key in the `dynamic_config` configmap